internal/bminventory/inventory_test.go

```diff
@@ -6539,6 +6539,7 @@ var _ = Describe("UpdateHostApproved", func() {
 	})
 
 	It("update approved value", func() {
+		mockEvents.EXPECT().AddEvent(gomock.Any(), clusterID, &hostID, models.EventSeverityInfo, fmt.Sprintf("Updated Approved to %t Host %s Cluster %s", true, hostID.String(), clusterID.String()), gomock.Any())
```

Comment:
please break the line and add `.Times(1)`

Reply:
Done